layers/+window-management/exwm/config.el

```diff
@@ -1,7 +1,7 @@
-(defvar exwm--terminal-command "xterm"
+(defvar exwm--terminal-command "gnome-terminal"
```

Comment:
Not everyone uses `gnome` when getting `exwm` I believe it's reasonable to assume that because `exwm` depends on `x` that people will have xterm installed.